From: Tommy Webb <tommy@calyxinstitute.org>
Date: Wed, 24 Jul 2024 01:34:21 +0000
Subject: Fix build after "Keep disabled WebViewMediaIntegrityApi"

This API was removed. Cromite itself doesn't actually build WebView, so
this problem isn't encountered there.

Chromium upstream change "Remove old Media Integrity code paths"
(commit 7dc55447f1d7fce3df99c4c6ab2429d28940e9fa) indicates that the
WebViewMediaIntegrityApi wasn't actually used and has been replaced
by the blink extension variant which is already disabled by the change.

Change-Id: I42cd39470e6093aad97141219a0ee7cc56ddafb3
---
 android_webview/common/aw_features.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/android_webview/common/aw_features.cc b/android_webview/common/aw_features.cc
--- a/android_webview/common/aw_features.cc
+++ b/android_webview/common/aw_features.cc
@@ -256,7 +256,6 @@ BASE_FEATURE(kCreateSpareRendererOnBrowserContextCreation,
              "CreateSpareRendererOnBrowserContextCreation",
              base::FEATURE_DISABLED_BY_DEFAULT);
 
-SET_CROMITE_FEATURE_DISABLED(kWebViewMediaIntegrityApi);
 SET_CROMITE_FEATURE_DISABLED(kWebViewMediaIntegrityApiBlinkExtension);
 }  // namespace features
 }  // namespace android_webview
-- 

